### PR TITLE
[Feature Request] kraken kubectl suppress ansible output by default.

### PR DESCRIPTION
**Pull request by alejandroEsc**
changed output of computed_kubectl to suppress ansible output, optionally allows you to turn it on via verbosity flag.

### CODE REVIEW

File: bin/max_k8s_version.sh [hunk 1]
```diff
@@ -23,7 +23,18 @@ source "${my_dir}/../lib/common.sh"
...
-ansible-playbook ${K2_VERBOSE} -i ansible/inventory/localhost \
-	ansible/max_k8s_version.yaml --extra-vars \
-	"${KRAKEN_EXTRA_VARS}${LOCAL_KEV}" \
-    || echo "max_version failed"
+function runcmd(){
+    ansible-playbook ${K2_VERBOSE} \
+        -i ansible/inventory/localhost \
+        ansible/max_k8s_version.yaml \
+        --extra-vars "${KRAKEN_EXTRA_VARS}${LOCAL_KEV}" \
+        || echo "max_version failed" > out
```
Comment (davidewatson) [inline bin/max_k8s_version.sh]: Why are we creating this file?

File: bin/max_k8s_version.sh [hunk 2]
```diff
@@ -23,7 +23,18 @@ source "${my_dir}/../lib/common.sh"
...
+}
+
+if [[ ! -z $K2_VERBOSE ]]; then
+    runcmd
+else
+    out=`runcmd`
```
Comment (davidewatson) [inline bin/max_k8s_version.sh]: Maybe I misunderstand, but why not redirect to /dev/null instead of capturing the output in a variable?

Comment (alejandroEsc) [reply bin/max_k8s_version.sh]: hmm ok that is a good point! doh, yeah ill update a new pr with that which includes one for ssh.

File: bin/max_k8s_version.sh (hunk 1 above)
Comment (alejandroEsc) [reply bin/max_k8s_version.sh]: not m preference. This is the pattern that kraken follows for getting the version of kubectl that is required based on k8s version. Probably would need to figure out how to get that info from the config file. Seriously we need to parse the schema into go already....

Comment (davidewatson) [reply bin/max_k8s_version.sh]: The result will be saved to ${OUTFILE}...

Comment (davidewatson) [reply bin/max_k8s_version.sh]: BTW, we may be closer to golang structs with https://github.com/samsung-cnct/jsonsvalidator

**Pull request merged**